packages/core/src/icon/icon.stories.ts

```diff
@@ -1,11 +1,56 @@
-import type { StoryObj } from '@storybook/web-components';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import type { IconStyle } from '@fortawesome/fontawesome-common-types';
 import { html } from 'lit';
 import './register.js';
+import { faPinata as falPinata } from '@fortawesome/pro-light-svg-icons';
+import { far } from '@fortawesome/pro-regular-svg-icons';
+import { faPinata as fasPinata } from '@fortawesome/pro-solid-svg-icons';
+import { faPinata as fatPinata } from '@fortawesome/pro-thin-svg-icons';
+import { Icon } from './icon.js';
+
+interface Props {
```

Comment:
So kind of nitpicking, but what I do is this:

```ts
interface Props<T = unknown> extends Pick<PageList<T>, 'items' | 'renderItem' | 'scopedElements'> {
  height?: string;
  styles?: string;
}
```

So pick the properties from the web component and then add any story-specific props you need.